applications/vessel-history/src/features/vessel-list-item/VesselListItem.tsx

```diff
@@ -9,7 +9,7 @@ import { useVesselsConnect } from 'features/vessels/vessels.hook'
 import { getFlagById } from 'utils/flags'
 import { getVesselAPISource } from 'utils/vessel'
 import { SHOW_VESSEL_API_SOURCE } from 'data/constants'
-import I18nDate, { formatI18nDate } from 'features/i18n/i18nDate'
+import I18nDate, { formatI18nDate, I18nDateAsString } from 'features/i18n/i18nDate'
```

Comment:
I'd only use `formatI18nDate`
```suggestion
import { formatI18nDate } from 'features/i18n/i18nDate'
```